deploy/Dockerfile-ppc64le

```diff
@@ -0,0 +1,13 @@
+FROM ppc64le/alpine:3.10
+MAINTAINER lysannef@us.ibm.com
+
```

Comment:
It looks like this doesn't include packages installed by the other Dockerfile.  Is there a reason these are omitted?

Reply:
Have added the packages, they do not affect the build of the Dockerfile.